src/task-actions.ts

```diff
@@ -88,8 +88,11 @@ export async function starknetCompileAction(args: TaskArguments, hre: HardhatRun
         });
     }
     if (hre.config.paths.cairoPaths) {
-        hre.config.paths.cairoPaths.forEach((path: string) => {
-            cairoPaths.push(path);
+        hre.config.paths.cairoPaths.forEach((cairPath: string) => {
+            if (!path.isAbsolute(cairPath)) {
+                cairPath = path.join(process.env.PWD, cairPath);
```

Comment:
Should it be `process.env.PWD` or `hre.config.paths.root` as we already do in L100? Also, `path.normalize` could be important.

Reply:
I think `hre.config.paths.root` is better for consistency's sake.